paper/paper.md

```diff
@@ -0,0 +1,233 @@
+config = SegmentedWindowConfig(indicators, change_metrics, [t[1]], [t[end]];
+    width_ind = length(residual) ÷ 2, whichtime = last, min_width_cha = 100)
+
+# Compute the metrics over sliding windows and their significance
+results = estimate_indicator_changes(config, data, time)
+signif = SurrogatesSignificance(n = 1000, tail = :right, rng = Xoshiro(1995))
+flags = significant_transitions(results, signif)
+
+# A minimal visualisation of the results
+fig = plot_changes_significance(results, signif)
+```
+
+We apply this code to data generated by a Ricker model presenting an abrupt transition
+at $t = 860$, which is used in the first tutorial of `ewstools` [@bury_ewstools_2023],
+the most recent software with functionalities that are similar to TransitionsInTimeseries.jl.
+The results are shown in [Fig. 1](@figure1) and display, as expected from CSD theory, an
+increase in both variance and AR1 coefficient, which is exactly the same as computed
+by `ewstools`. However, calling `signif.pvalues` shows that the increase in variance is not
+significant ($p = 0.284$), whereas the increase in AR1 coefficient is ($p = 0.001$).
+
+![Output of plotting function in usage example.\label{fig: fig1}](figures/figure1.png)
+
+We believe that a concise and unambiguous code will greatly reduce the programming effort of
+many researchers and ease the code reviewing process. Finally, the code documentation
+provides additional examples, showcasing that the simplicity of the code also applies to
+real-world applications.
+
+## Performance
+
+TransitionsInTimeseries.jl is written in Julia, which offers both a simple syntax and good
+performance. Additionally, the run-time of all performance-relevant steps have been
+optimized. Significance testing can be the main bottleneck and was therefore parallelised, 
+which simply requires the user to activate more threads in their Julia REPL. This is of
+particular importance since the transition analysis is sometimes performed on 10,000+
+timeseries [@ben-yami_uncertainties_2023], which can last hours with a serial code that is
+not optimized.
+
+## Reproducibility
+
+Some steps of a transition analysis involve random number generators, which need 
+to be handled with care in parallelized codes. This is done in TransitionsInTimeseries.jl, 
+which offers the possibility of seeding a random number generator by using the 
+keywordargument `rng`, as done in the example shown above. Furthermore, 
+TransitionsInTimeseries.jl follows the guidelines of semantic versionning which, 
+along with Julia's integrated package manager, ensures that the same code is used for both 
+results generation and peer reviewing.
+
+## Reliability
+
+In high-impact context mentioned above, errors are to be avoided with particular care.
+TransitionsInTimeseries.jl is therefore tested via continuous integration on a large test
+suite, thus providing a reliable research framework. Furthermore, a centralised code base
```

Comment:
If we talk about the centralized code here, in the "statement of need" section we should say that "our vision with this software is to become the established standard, widely used by most academics working on transitions in timeseries".

Reply:
Yes! I rephrased this in the new commits.